src/main/java/com/onarandombox/MultiverseCore/commands/GamerulesCommand.java

```diff
@@ -68,7 +74,7 @@ public void runCommand(CommandSender sender, List<String> args) {
                 gameRules.append(ChatColor.WHITE).append(", ");
             }
             gameRules.append(ChatColor.AQUA).append(gameRule).append(ChatColor.WHITE).append(": ");
-            gameRules.append(ChatColor.GREEN).append(world.getGameRuleValue(gameRule));
+            gameRules.append(ChatColor.GREEN).append(world.getGameRuleValue(GameRule.getByName(gameRule)));
```

Comment:
This could potentially cause issues if the enum values ever change as we'll have persisted them differently. It's an unlikely thing but something to consider.

Reply:
I don't understand the concern here, as `gameRule` comes from the Bukkit World's list of gamerules. Could you clarify?

Reply:
Ah, nevermind then :)